js/irma-backend/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.5] - 2022-01-24
+### Security
+- Updated depency `eventsource` to v2.0.2
+### Added
+- package-lock.json
```

Comment:
I don't really understand why `package-lock.json` is added? It was already there if I'm right.